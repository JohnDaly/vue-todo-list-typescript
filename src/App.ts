// External Dependencies
import { Component, Vue } from 'vue-property-decorator';

// Internal Dependencies
import Checklist from './components/Checklist/Checklist.vue';

@Component({
  components: { Checklist },
})
export default class App extends Vue {}
