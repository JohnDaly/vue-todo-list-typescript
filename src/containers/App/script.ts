// External Dependencies
import { Component, Vue } from 'vue-property-decorator';

// Internal Dependencies
import Checklist from '@/components/Checklist/index.vue';

@Component({
  components: { Checklist },
})
export default class App extends Vue {}
