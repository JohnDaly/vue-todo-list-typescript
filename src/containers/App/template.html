<div id='app'>
    <Checklist title='Vue.js Todo List' />
</div>