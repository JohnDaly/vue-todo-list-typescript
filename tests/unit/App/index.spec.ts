// External Dependencies
import { shallowMount } from '@vue/test-utils';

// Internal Dependencies
import App from '@/containers/App/index.vue';

describe('App.vue', () => {
    it('should render properly', () => {
        const wrapper = shallowMount(App, {});
        expect(wrapper).toMatchSnapshot();
    });
});
