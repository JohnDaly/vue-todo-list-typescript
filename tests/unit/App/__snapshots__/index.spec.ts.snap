// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App.vue should render properly 1`] = `
<div id="app">
  <checklist-stub title="Vue.js Todo List"></checklist-stub>
</div>
`;
